tools/mtouch/mtouch.cs

```diff
@@ -294,255 +294,6 @@ public static IList<string> GetAotArguments (Application app, string filename, A
 			return args;
 		}
 
-		static string EncodeAotSymbol (string symbol)
-		{
-			var sb = new StringBuilder ();
-			/* This mimics what the aot-compiler does */
-			foreach (var b in System.Text.Encoding.UTF8.GetBytes (symbol)) {
-				char c = (char) b;
-				if ((c >= '0' && c <= '9') ||
-					(c >= 'a' && c <= 'z') ||
-					(c >= 'A' && c <= 'Z')) {
-					sb.Append (c);
-					continue;
```

Comment:
Well we are moving code, maybe chaging to Char.Is* methods is not a good idea since we already had this logic.